config-local.php-dist

```diff
@@ -6,11 +6,13 @@
  */
 
 // LDAP
-define('LDAP_HOST', 'pm-ns.mozilla.org');
+define('LDAP_HOST', 'DEFAULT_LDAP_HOST');
+define('LDAP_EXCLUDE', '');  // '(!(defaultAttribute=defaultValue))'
```

Comment:
This allows us to provide additional LDAP filters that are AND'd into the search filter, for us to perform server-side excludes beyond the publicly-documented employeeType=DISABLED filter. We cannot make these filters public, so I extended the code to accept it through config-local and provided an example of one such filter for an anonymous attribute and value. (The actual LDAP_EXCLUDE we're using is in an email to your work address, but please do not publish it here or anywhere.)